test/end-to-end-tests/run.sh

```diff
@@ -35,5 +35,6 @@ trap 'handle_error' ERR
 if [ $has_custom_app -ne "1" ]; then
     ./element/start.sh
 fi
-node start.js $@
+yarn build
```

Comment:
I've added a build step to run.sh - this seems reasonable as it only takes a few seconds on my machine, but maybe we'll want separate control over it at some point